docs/userdoc/SimulationLoop.md

```diff
@@ -10,6 +10,11 @@ AutoPas assumes that any (short-range) particle simulation boils down to three h
 ## Code Example
 A conceptual example of a primary simulation loop:
 ```c++
+
+// Initial force calculation
+YourFunctor functor();                       // User code
+autopas->iteratePairwise(&functor);          // Tuning and parallelization happens here
+
```

Comment:
I'd not add this here. 
1. The Functor you create here would shadow persist for the whole simulation ;)
2. Whether this is necessary or not depends on the kind of simulation and this document is intended to serve as a generic example of the necessary steps and where AutoPas is involved. Thus I'd keep everything else to a minimum to retain focus.